### DISCUSSION TITLE
Keep skin/beatmap archives after importing them

### DISCUSSION THREAD

**Discussion by facespkz** [ppy/osu#22920] [Ideas]
This is something that sort of bugged me on stable, but I'd like to have an option to keep the archives of skins and songs for ease of backup. It helps with songs that get un-submitted so that you can't download them anymore.

**Comment by bdach** [comment]
98% of people *want* the archives to be gone after import to not double up disk space. If you wish to keep backups then surely an option is to ctrl-c ctrl-v before importing?

**Reply by facespkz** [reply]
I could copy and paste the archive away from my sync folder, but... it would be more time efficient if the game just didn't delete it in the first place. Besides, I've actually imported something on accident before. I could also directly download it from my cloud storage's frontend, and it would be just like downloading the file from a beatmap's page.

This would basically be a toggle, on or off, to disable the bit that says to delete the file when the game is done importing. With bugs like #17485 and #21942, it would probably be a good idea for some to have the original archive in the meantime.

**Reply by peppy** [reply]
I'm fine with this being a toggle which defaults to the current behaviour.
